docs/grid/your_first_grid.md

```diff
@@ -42,10 +41,9 @@ Now we can configure our first grid:
  ### **Note**
 
 Remember that a grid is *the way objects of a desired entity are
-displayed on its index view*. Therefore only fields that are useful
+displayed on its index view*. Therefore, only fields that are useful
 for identification of objects are available - only `string` and `twig`
-type. Then even though a Supplier has also a description field, it is
-not needed on index and can't be displayed here.
+types. Although a Supplier also has a description field, it is not needed on the index and cannot be displayed here.
```

Comment:
Technically we can display `description` field in the grid, but won't as it's not needed.

```suggestion
types. Although a Supplier also has a _description_ field, it is not needed on the index and won't be displayed here.
```